scripts/install_moodle.sh

```diff
@@ -179,40 +204,41 @@ apt-get autoremove -y
 echo_info "Done."
 
 ###############################################################################
-echo_title "Setup SMTP Relay."
+echo_title "Setup SMTP Server."
 ###############################################################################
-echo_action "Adding SMTP Relay Private IP address in ${hostsFilePath}..."
-if ! grep -q "${parameters[smtpRelayFqdn]}" $hostsFilePath; then
-    echo -e "\n# Redirect SMTP Relay FQDN to Private IP Address.\n${parameters[smtpRelayPrivateIp]}\t${parameters[smtpRelayFqdn]}" >> $hostsFilePath
-    echo_info "Done."
+echo_action "Adding SMTP Server Private IP address in ${HOSTS_FILE_PATH}..."
+if ! grep -q "${parameters[smtpRelayFqdn]}" ${HOSTS_FILE_PATH}; then
+  echo -e "\n# Redirect SMTP Server FQDN to Private IP Address.\n" >> ${HOSTS_FILE_PATH}
+  echo -e "${parameters[smtpRelayPrivateIp]}\t${parameters[smtpRelayFqdn]}" >> ${HOSTS_FILE_PATH}
+  echo_info "Done."
 else
-    echo_info "Skipped: ${hostsFilePath} file already set up."
+  echo_info "Skipped: ${HOSTS_FILE_PATH} file already set up."
 fi
 
 ###############################################################################
 echo_title "Update PHP config."
 ###############################################################################
-echo_action "Updating upload_max_filesize and post_max_size settings in ${phpIniFilePath}..."
-sed -i "s/upload_max_filesize.*/upload_max_filesize = 2048M/" $phpIniFilePath
-sed -i "s/post_max_size.*/post_max_size = 2048M/" $phpIniFilePath
+echo_action "Updating upload_max_filesize and post_max_size settings in ${PHP_INI_FILE_PATH}..."
+sed -i "s/upload_max_filesize.*/upload_max_filesize = 2048M/" ${PHP_INI_FILE_PATH}
+sed -i "s/post_max_size.*/post_max_size = 2048M/" ${PHP_INI_FILE_PATH}
 echo_info "Done."
 
 ###############################################################################
 echo_title "Update Apache config."
 ###############################################################################
-echo_action "Updating Apache default site DocumentRoot property in ${apache2SitesEnabledDefaultFilePath}..."
-if ! grep -q "${moodleDocumentRootDirPath}" $apache2SitesEnabledDefaultFilePath; then
-    escapedApache2DefaultDocumentRootDirPath=$(sed -E 's/(\/)/\\\1/g' <<< ${apache2DefaultDocumentRootDirPath})
-    escapedMoodleDocumentRootDirPath=$(sed -E 's/(\/)/\\\1/g' <<< ${moodleDocumentRootDirPath})
-    sed -i -E "s/DocumentRoot[[:space:]]*${escapedApache2DefaultDocumentRootDirPath}/DocumentRoot ${escapedMoodleDocumentRootDirPath}/g" $apache2SitesEnabledDefaultFilePath
-    echo_info "Done."
+echo_action "Updating Apache default site DocumentRoot property in ${APACHE2_SITE_ENABLED_DEFAULT_FILE_PATH}..."
+if ! grep -q "${MOODLE_DOCUMENT_ROOT_DIR_PATH}" ${APACHE2_SITE_ENABLED_DEFAULT_FILE_PATH}; then
+  # escapedApache2DefaultDocumentRootDirPath=$(sed -E 's/(\/)/\\\1/g' <<< ${apache2DefaultDocumentRootDirPath})
+  # escapedMoodleDocumentRootDirPath=$(sed -E 's/(\/)/\\\1/g' <<< ${moodleDocumentRootDirPath})
```

Comment:
Are these necessary or they can be removed?

Reply:
???? It's not in the code...